index.js

```diff
@@ -2,11 +2,16 @@ const path = require('path');
 const core = require('@actions/core');
 const tmp = require('tmp');
 const fs = require('fs');
+const {ECS} = require('@aws-sdk/client-ecs');
 
 async function run() {
   try {
+    const ecs = new ECS({
+      customUserAgent: 'amazon-ecs-render-task-definition-for-github-actions'
+    });
+
     // Get inputs
-    const taskDefinitionFile = core.getInput('task-definition', { required: true });
+    const taskDefinition = core.getInput('task-definition', { required: false });
```

Comment:
This variable name should be kept to what it was to start with (`taskDefinitionFile`) because changing this might introduce a breaking change for users.